src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,56 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
-    public void getStatistic(String fromFileName, String toFileName) {
+    private static final String VALUE_BUY = "buy";
+    private static final String VALUE_SUPPLY = "supply";
+    private static final String VALUE_RESULT = "result";
+    private static final String COMMA_SEPARATOR = ",";
 
+    public void getStatistic(String fromFileName, String toFileName) {
+        String fileRead;
```

Comment:
Method is too long to be easily read and understood.
https://mate-academy.github.io/jv-program-common-mistakes/java-core/builder-file/work-with-file#divide-program-logic

Please split logic to 3 separate private methods (to read from file, to generate report, to write report to file). And then call these 3 private methods in public method